src/ResourceManagement/OperationalInsights/OperationalInsights.Test/ScenarioTests/SearchOperationsTests.Scenario.cs

```diff
@@ -0,0 +1,231 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.OperationalInsights;
+using Microsoft.Azure.Management.OperationalInsights.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using OperationalInsights.Tests.Helpers;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace OperationalInsights.Test.ScenarioTests
+{
+    public class SearchOperationsTests : TestBase
+    {
+
+        [Fact]
+        public void CanGetSearchResultsAndUpdate()
+        {
+            using (MockContext context = MockContext.Start(GetType().FullName))
+            {
+                var client = TestHelper.GetOperationalInsightsManagementClient(this, context);
+
+                // Rasha's workspace is the only one in int with cold data, which is necessary to test the update search method
+                string resourceGroupName = "OI-Default-East-US";
```

Comment:
I changed the comments to explain why we are doing this. Basically, we can't just create a brand new workspace to test in this case because there are no ARM APIs to ingest data to a workspace to test search. So we have to rely on a workspace that already had data ingested   beforehand. I explained in the comments that any workspace with data ingested should work fine for this test. 